ttl/ttlworker/job_manager_integration_test.go

```diff
@@ -123,3 +124,58 @@ func TestFinishJob(t *testing.T) {
 
 	tk.MustQuery("select table_id, last_job_summary from mysql.tidb_ttl_table_status").Check(testkit.Rows("2 {\"total_rows\":0,\"success_rows\":0,\"error_rows\":0,\"total_scan_task\":1,\"scheduled_scan_task\":0,\"finished_scan_task\":0,\"scan_task_err\":\"\\\"'an error message contains both single and double quote'\\\"\"}"))
 }
+
+func TestTTLAutoAnalyze(t *testing.T) {
+	ttlworker.SetResizeWorkersInterval(time.Second)
+	ttlworker.SetUpdateInfoSchemaCacheInterval(time.Second)
+	ttlworker.SetUpdateTTLTableStatusCacheInterval(time.Second)
```

Comment:
I think it can be done by `failpoint.Enable('xxxx', 'return(time.Second)')`

Reply:
Cool. Let me try

Reply:
Done.

A little problem is that the failpoint cannot parse `time.Second`, I used `1000000000` instead :cry: .

Reply:
@YangKeao Maybe `fmt.Sprintf("return(%d)", time.Second)`, or just `return("1s")`?

Reply:
Done. PTAL @xhebox